src/js/components/Menu/__tests__/Menu-test.js

```diff
@@ -522,4 +535,27 @@ describe('Menu', () => {
     );
     expect(component.toJSON()).toMatchSnapshot();
   });
+
+  test('custom theme with default button', () => {
```

Comment:
Thanks for enhancing the testing! Can you try to separate the testing into a separate PR? After the testing PR will be merged it will be easier to understand the impact of changes this PR is suggesting.

Reply:
Yes, will do now.